src/Style/TerminusStyle.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Pantheon\Terminus\Style;
+
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Terminus\Models\Workflow;
+
+class TerminusStyle extends SymfonyStyle
+{
+    const NORMAL_PROGRESS_FORMAT = " <bg=blue;fg=black> %message% </>\n <info>Progress: %bar% <info>%percent:3s%%</info> \n <info>Elapsed: %elapsed:6s% \n Estimated Time Remaining: %estimated:-6s%</info>";
```

Comment:
We might want to make this a property rather than a class constant too, allowing for getting/setting